examples/Clients/Greeter/Program.cs

```diff
@@ -31,16 +33,13 @@ class Program
         static async Task Main(string[] args)
         {
             // Server will only support Https on Windows and Linux
-            var credentials = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? ChannelCredentials.Insecure : ClientResources.SslCredentials;
-            var channel = new Channel("localhost:50051", credentials);
-            var client = new Greeter.GreeterClient(channel);
+            var certificate = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? null : new X509Certificate2(Resources.ServerPFXPath, "1111");
+            var client = GrpcClientFactory.Create<Greeter.GreeterClient>("https://localhost:50051", certificate: certificate);
```

Comment:
Undo this. The samples should use the C Core client until duplex streaming is ready. Once that is done we can recommend people to use it.